### PR TITLE
do not do jumped takeoff if already in air

### PR DESCRIPTION
**Pull request by tumbili**
Hotfix for takeoff. Went testing today and realised that drone would fall out of the sky if mission with takeoff is entered in air. I was able to reproduce this also in simulation.

### CODE REVIEW

File: src/modules/mc_pos_control/mc_pos_control_main.cpp [hunk 1]
```diff
@@ -1378,7 +1385,6 @@ MulticopterPositionControl::task_main()
 
 					if (_takeoff_jumped) {
 						_vel_sp(2) = -_params.tko_speed;
-						_vel_sp_prev(2) = -_params.tko_speed;
```
Comment (tumbili) [inline src/modules/mc_pos_control/mc_pos_control_main.cpp]: I removed this because it lead to the thrust setpoint being able to jump.

**Comment by julianoes** [comment]
I can follow your changes. I would say it's for the better ;)

**Comment by AndreasAntener** [comment]
This probably fixes https://github.com/PX4/Firmware/issues/3595

**Comment by AndreasAntener** [comment]
Applied to master

**Pull request closed without merging**